src/components/App/viewmodel.ts

```diff
@@ -293,7 +293,16 @@ export class AppViewModel {
     };
 
     this.sharerConnection.ontrack = (event) => {
-      this.video.current!.srcObject = event.streams[0];
+      if(event.track.kind == "video") {
+        this.video.current!.srcObject = event.streams[0];
+      } else if (event.track.kind == "audio") {
+        const audio_elem = document.createElement("audio");
+        audio_elem.srcObject = event.streams[0];
+        //alert(1);
+        audio_elem.play();
```

Comment:
Will not succeed if user has not interacted with the page yet.

TODO: If play ('s promise) fails, prompt user to interact with the page (e.g. click anywhere in the page), and then attempt to play again

Reply:
fixed.